OutOfSchool/OutOfSchool.BusinessLogic/Models/Teachers/TeacherDTO.cs

```diff
@@ -11,19 +12,19 @@ public class TeacherDTO
 
     [Required(ErrorMessage = "First name is required")]
     [DataType(DataType.Text)]
-    [MaxLength(60)]
-    [RegularExpression(@"^([^0-9]*)$", ErrorMessage = "First name cannot contains digits")]
+    [MaxLength(Constants.NameMaxLength)]
+    [CustomUkrainianName(ErrorMessage = "First name cannot contains digits")]
```

Comment:
(Message wording) Message text should be changed as you're no longer checking for digits. Same in other properties